wire-ios-sync-engine/Source/Synchronization/Strategies/UserClientEventConsumer.swift

```diff
@@ -34,12 +34,13 @@ public class UserClientEventConsumer: NSObject, ZMEventAsyncConsumer {
         managedObjectContext: NSManagedObjectContext,
         clientRegistrationStatus: ZMClientRegistrationStatus,
         clientUpdateStatus: ClientUpdateStatus,
-        resolveOneOnOneConversations: ResolveOneOnOneConversationsUseCaseProtocol
+        resolveOneOnOneConversations: any ResolveOneOnOneConversationsUseCaseProtocol
```

Comment:
What is the difference when adding `any`?

Reply:
The best documentation is actually the original proposal with the downsides and solution approach:
https://github.com/apple/swift-evolution/blob/main/proposals/0335-existential-any.md#introduction

Short summary I copied from multiple paragraphs:
> A plain protocol name in type context means an existential type.
> Existential types in Swift have significant limitations and performance implications.
> Existential types are also significantly more expensive than using concrete types.
> The cost of using existential types should not be hidden, and programmers should explicitly opt into these semantics.
> I propose to make existential types syntactically explicit in the language using the any keyword.

It got introduced as optional in Swift 5.6, the plan was to enforce it by the compiler with Swift 6, but it seems that it was deferred to a later release.